### PR TITLE
Added sky adapter service

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: src/app/public/modules/adapter-service/adapter.service.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  'iframe',
+  'object',
+  'embed',
+  '*[tabindex]:not([tabindex=\'-1\'])',
+  '*[contenteditable=true]'
+].join(', ');
+/* tslint:enable */
+
+@Injectable()
+export class SkyAdapterService {
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/adapter-service/adapter.service.ts]: I think we might need a better name. I totally get that this is an adapter but I'm wondering if we will get confused down the road with what this is vs the other adapters we have. Thoughts?

File: src/app/public/modules/adapter-service/adapter.service.ts [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  Renderer2,
+  RendererFactory2
+} from '@angular/core';
+
+import {
+  SkyMediaBreakpoints
+} from '../media-query';
+
+/* tslint:disable */
+const SKY_TABBABLE_SELECTOR = [
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/adapter-service/adapter.service.ts]: Why do we need to disable linting here?

File: src/app/public/modules/adapter-service/adapter.service.ts [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+  'object',
+  'embed',
+  '*[tabindex]:not([tabindex=\'-1\'])',
+  '*[contenteditable=true]'
+].join(', ');
+/* tslint:enable */
+
+@Injectable()
+export class SkyAdapterService {
+  private renderer: Renderer2;
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/adapter-service/adapter.service.ts]: Could we place an empty line above this? Personal preference :)

File: src/app/public/modules/adapter-service/adapter.service.ts [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    this.renderer.addClass(nativeEl, newClass);
+  }
+
+  /**
+   * This method temporarily enables/disables pointer events.
+   * This is helpful to prevent iFrames from interfering with drag events.
+   *
+   * @param {boolean} enable - Set to `true` to enable pointer events. Set to `false` to disable.
+   */
+  public toggleIframePointerEvents(enable: boolean): void {
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/adapter-service/adapter.service.ts]: Do we do this somewhere outside of flyouts? Wondering if we need to pull this out? Either way si fine.

File: src/app/public/modules/adapter-service/adapter.service.ts (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/adapter-service/adapter.service.ts]: `SkyCoreAdapterService`?

File: src/app/public/modules/adapter-service/adapter.service.ts (hunk 4 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/adapter-service/adapter.service.ts]: Split-view will also need it, as it has dragging.

File: src/app/public/modules/adapter-service/adapter.service.ts (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/adapter-service/adapter.service.ts]: Leftover from previous code. Removed it.

File: src/app/public/modules/adapter-service/adapter.service.ts (hunk 3 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/adapter-service/adapter.service.ts]: Done.

File: src/app/public/modules/adapter-service/adapter.service.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/adapter-service/adapter.service.ts]: Maybe? Lol. I think that is better than what is currently here.

Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/adapter-service/adapter.service.ts]: Done.

File: src/app/public/modules/adapter-service/adapter.module.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+import {
+  SkyCoreAdapterService
+} from './adapter.service';
+
+@NgModule({
+  providers: [
+    SkyCoreAdapterService
+  ]
+})
+export class SkyAdapterModule { }
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/adapter-service/adapter.module.ts]: I think we should rename the module here too.

Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/adapter-service/adapter.module.ts]: Ah, good catch. Done.

**Pull request merged**